README.md

```diff
@@ -165,6 +149,22 @@ options:
   --hdop-max HDOP_MAX  reject GPS messages where hdop exceeds this limit, default 100.0 (no limit)
 ~~~
 
+### show_types.py
+
+~~~
+$ show_types.py --help
+usage: show_types.py [-h] [-r] paths [paths ...]
```

Comment:
Is each argument multiple paths or can multiple paths be given on the command line?  I suspect it's probably
```
usage: show_types.py [-h] [-r] path [path ...]
```

Reply:
You are right. I'll make this change to all tools.